how-to-use-this-module/terrafile.tf

```diff
@@ -1,6 +1,12 @@
+provider "google" {
+  region = "us-central1"
+    }
+  
+
+
 module "network_gcp" {
-  source  = "./.."
-  project = "nomedoprojeto"
+  source   = "./.."
+  project  = "nomedoprojeto"
```

Comment:
Nessa linha temos o seguinte problema. Como isso aqui é estático, precisamos criar o projeto com nome "nomedoprojeto" em nossa conta pra validar esse terraform. 

IMHO (In My Humble Opinion - Em minha humilde opinião) eu acho que temos dois caminhos:

- Colocar um nome mais genérico para testes (Caminho mais fácil. Precisaria trocar a documentação pra deixar claro isso)
- Colocar o nome do projeto como variável do terraform, e carregar isso como variável de ambiente. (Adiciona complexidade para quem vai testar o módulo local, mas não muito. Eu gosto muito dessa opção aqui, ainda mais se colocarmos pra o terraform rodar em docker, com make, fica tranquilo assim.) 
- Criar uma pasta focada para teste, com valores estáticos para teste (Eu gosto menos dessa opção)